src/app/courses/components/add-course/add-course.component.html

```diff
@@ -0,0 +1,82 @@
+<section class="add-course-container">
+    <header class="add-course-header">
+        <h2 class="add-course-title">add course</h2>
+        <div class="separator"></div>
+        <!-- hidden for now -->
+        <!-- <div class="close-btn" role=button>
+            <fa-icon icon="times"></fa-icon>
+        </div> -->
+    </header>
+    <form class="add-course-form">
+        <ul class="form-sections">
+            <li class="course-title-container">
+                <label for="course-title-id">title</label>
+                <input
+                    type="text"
+                    id="course-title-id"
+                    class="course-title"
+                    placeholder="Enter a course name"
+                    [(ngModel)]="courseTitle"
+                    name="title">
+            </li>
+            <li class="course-description-container">
+                <app-input-text></app-input-text>
+            </li>
+            <li class="course-meta-info-container">
+                <div class="course-date-container">
+                    <label for="course-date-id">date</label>
+                    <input
+                        type="date"
+                        id="course-date-id"
+                        placeholder="Enter a course date"
+                        class="course-date">
```

Comment:
It must be separate component

Reply:
Indeed, turns out I mixed up this requirement with the `textarea` component, but is it really important right now for the change detection task?

Reply:
Change detection task consists of 2 parts. "Implementation of change detection" and "Course add/edit page markup + basic structure". And this note is relative to the last one. Also as per marks there is a criteria for mark 4 "Added implementations for all components, with some major issues". So there is no component, it is my explanation to your 4,5 mark.

Reply:
Ok, no problem at all.